Showcase/Showcase.md

```diff
@@ -115,7 +116,6 @@ With aligned columns
 
 | Name                 | CommandType | Version |
 |:-------------------- |:-----------:| -------:|
-| New-MDAlert          | Function    | 0.0     |
```

Comment:
Why was removed?

Reply:
This should be good now.

Reply:
How was this fixed? Because the script either works or not and it shouldn't be a manual fix. If the script has an issue, then we need to resolve it.

Reply:
I am clueless, I suspect there was already in memory module MarkdownPS which did not have New-MDAlert function yet. But then again it should reload it completely when it imports module.ps1. 
I just ran it again (with clean -noprofile terminal) and this time it worked.